lib/athena/index.js

```diff
@@ -27,7 +28,8 @@ class AthenaQueryClient {
         QueryString: sql,
         ResultConfiguration: {
           OutputLocation: `s3://${this.s3Bucket}`
-        }
+        },
+        WorkGroup: `${this.workgroup}`
```

Comment:
can just make this `WorkGroup: this.workgroup`

Reply:
good catch!